src/components/Swap/ReverseButton.tsx

```diff
@@ -10,7 +10,7 @@ import Row from '../Row'
 const ReverseRow = styled(Row)`
   left: 50%;
   position: absolute;
-  transform: translate(-50%, -50%);
+  transform: translate(-50%, -60%);
```

Comment:
Why is this necessary? Will this not off center it?

Reply:
with the modified asymmetric padding values we're using now this is necessary to keep the button centered